packages/editor-ui/src/plugins/telemetry/index.ts

```diff
@@ -55,19 +55,30 @@ class Telemetry {
 		this.pageEventQueue = [];
 	}
 
-	init(options: ITelemetrySettings, instanceId: string, logLevel?: ILogLevel) {
+	init(options: ITelemetrySettings, {instanceId, logLevel, userId}: {instanceId: string, logLevel?: ILogLevel, userId?: string}) {
 		if (options.enabled && !this.telemetry) {
 			if(!options.config) {
 				return;
 			}
 
 			const logging = logLevel === 'debug' ? { logLevel: 'DEBUG'} : {};
 			this.loadTelemetryLibrary(options.config.key, options.config.url, { integrations: { All: false }, loadIntegration: false, ...logging});
-			this.telemetry.identify(instanceId);
+			this.identify(instanceId, userId);
 			this.flushPageEvents();
 		}
 	}
 
+	identify(instanceId: string, userId?: string) {
+		this.telemetry.reset();
```

Comment:
to identify new users https://www.rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/#calling-the-identify-api-with-a-new-userid
will not reset anonymous id